### PR TITLE
[DOC] change clip grad api, test=develop

### PR DESCRIPTION
**Pull request by qili93**
Change doc of 3 Clip apis.

Paddle PR: https://github.com/PaddlePaddle/Paddle/pull/27767

### CODE REVIEW

**Comment by qili93** [comment]
image

image

image

**Comment by qili93** [comment]
image

image

image

File: doc/paddle/api/paddle/fluid/clip/ClipGradByValue_cn.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
+.. _cn_api_fluid_clip_ClipGradByValue
+
+ClipGradByValue
+-------------------------------
+
+.. py:class:: paddle.nn.ClipGradByValue(max, min=None, need_clip=None)
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/fluid/clip/ClipGradByValue_cn.rst]: 没有need_clip

File: doc/paddle/api/paddle/fluid/clip/ClipGradByGlobalNorm_cn.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
+.. _cn_api_fluid_clip_ClipGradByGlobalNorm:
+
+ClipGradByGlobalNorm
+-------------------------------
+
+.. py:class:: paddle.nn.ClipGradByGlobalNorm(clip_norm, group_name='default_group', need_clip=None)
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/fluid/clip/ClipGradByGlobalNorm_cn.rst]: 应该只有clip_norm吧 没有group_name、need_clip

File: doc/paddle/api/paddle/fluid/clip/ClipGradByGlobalNorm_cn.rst [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+            
+其中：
+
+.. math::            
+            \\global\_norm=\sqrt{\sum_{i=0}^{n-1}(l2norm(t\_list[i]))^2}\\
+
+
+参数:
+ - **clip_norm** (float) - 所允许的范数最大值
+ - **group_name** (str, optional) - 剪切的组名
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/fluid/clip/ClipGradByGlobalNorm_cn.rst:34]: 没有group_name吧

File: doc/paddle/api/paddle/fluid/clip/ClipGradByNorm_cn.rst [hunk 4]
```diff
@@ -0,0 +1,59 @@
+.. _cn_api_fluid_clip_ClipGradByNorm:
+
+ClipGradByNorm
+-------------------------------
+
+.. py:class:: paddle.nn.ClipGradByNorm(clip_norm, need_clip=None)
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/fluid/clip/ClipGradByNorm_cn.rst]: 没有need_clip

File: doc/paddle/api/paddle/fluid/clip/ClipGradByValue_cn.rst (hunk 1 above)
Comment (qili93) [reply doc/paddle/api/paddle/fluid/clip/ClipGradByValue_cn.rst]: Done

File: doc/paddle/api/paddle/fluid/clip/ClipGradByGlobalNorm_cn.rst (hunk 2 above)
Comment (qili93) [reply doc/paddle/api/paddle/fluid/clip/ClipGradByGlobalNorm_cn.rst]: 确认过，保留group_name，删除need_clip

File: doc/paddle/api/paddle/fluid/clip/ClipGradByGlobalNorm_cn.rst (hunk 3 above)
Comment (qili93) [reply doc/paddle/api/paddle/fluid/clip/ClipGradByGlobalNorm_cn.rst:34]: 确认过，保留group_name

File: doc/paddle/api/paddle/fluid/clip/ClipGradByNorm_cn.rst (hunk 4 above)
Comment (qili93) [reply doc/paddle/api/paddle/fluid/clip/ClipGradByNorm_cn.rst]: Done

**Review by TCChenlong — APPROVED** [review]
LGTM

**Review by hong19860320 — APPROVED** [review]
LGTM

**Pull request merged**